Nos/Views/Components/FlagOptionPicker.swift

```diff
@@ -0,0 +1,105 @@
+import SwiftUI
+/// Displays a list of selectable flag options
+struct FlagOptionPicker: View {
+    @Binding private var selectedOption: FlagOption?
+    var options: [FlagOption]
+    var title: String
+    var subtitle: String?
+
+    init(selectedOption: Binding<FlagOption?>, options: [FlagOption], title: String, subtitle: String?) {
+        self._selectedOption = selectedOption
+        self.options = options
+        self.title = title
+        self.subtitle = subtitle
+    }
+
+    var body: some View {
+        VStack(alignment: .leading) {
+            HeaderView(text: title)
```

Comment:
I wonder if we could put this into a navigation bar or title bar (I forget what it's called in SwiftUI) to more closely match the design.

Reply:
This was meant to address these two strings. There is another ticket to address the navigation bar. I wonder how I didn't see that. Thanks for catching it.
<img width="383" alt="Screenshot 2024-09-19 at 3 15 21 PM" src="https://github.com/user-attachments/assets/802440b8-ccc3-4444-ae77-39b2724cd7d2">